election/views.py

```diff
@@ -1,9 +1,8 @@
-import os
+import os,requests
```

Comment:
It seems that you are not using a python formatter. The PEP8 states we shouldn't import 2 libs over the same line